blocks-common/b-page/__csp/b-page__csp.bemhtml

```diff
@@ -0,0 +1,73 @@
+block b-page, elem csp {
+
+    bem: false
+
+    tag: 'meta'
+
+    attrs: {
+        'http-equiv': 'Content-Security-Policy',
+        'content': Object.keys(this.ctx.policies).map(function(name) {
```

Comment:
@alexbaumgertner Let's add deps from `i-ecma__object`, in 3.x it could be an issue.